### PR TITLE
Added: Mark theme as deprecated

### PR DESCRIPTION
**Pull request by math-GH**
While I update all themes and look into the little survey (#4295) I see 3 themes, that are not used often:
- BlueLagoon
- Flat
- Screwdriver

It would make sense to reduce our offered themes by this 3 themes. Some new themes were added in 1.20.

(IMHO: this 3 theme's designs are out of date)

Changes proposed in this pull request:

- option to mark a theme as deprecated via `manifest.json`
- marked the 3 themes as deprecated
- show the deprecation status on the reading configuration
- documentation improved (added the new themes, added the deprecation information)

How to test the feature manually:

1. see the theme slider

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by math-GH** [comment]
Maybe we can ship it with 1.20.1 already to inform the users as early as possible

**Comment by Alwaysin** [comment]
I don't think that those 30 votes on github can be taken as representative to the whole userbase.

**Comment by Alwaysin** [comment]
Just a thought but maybe we could as with the extensions not ship all the themes by default but put them on a separate repository.

P. S.: I do use flat from time to time!

**Comment by Alkarex** [comment]
Let's delay that to 1.21 so that we can have this discussion, avoid adding new translations at the last minute, and avoid taking regressions risks :-)

**Comment by math-GH** [comment]
How to go forward here?

**Comment by math-GH** [comment]
@Alkarex How do we want to proceed here?

File: app/i18n/fr/conf.php [hunk 1]
```diff
@@ -48,7 +48,13 @@
 			'timeout' => 'Temps d’affichage de la notification HTML5',
 		),
 		'show_nav_buttons' => 'Afficher les boutons de navigation',
-		'theme' => 'Thème',
+		'theme' => array(
+			'_' => 'Thème',
+			'deprecated' => array(
+				'_' => 'Deprecated',	// TODO
+				'description' => 'This theme is deprecated and will be not available anymore in a <a href=https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#theme target=_blank>future release of FreshRSS</a>',	// TODO
```
Comment (Alkarex) [inline app/i18n/fr/conf.php]:
```suggestion
				'_' => 'Obsolète',
				'description' => 'Ce thème est obsolète et sera supprimé dans une <a href="https://freshrss.github.io/FreshRSS/fr/users/05_Configuration.html#th%C3%A8me" target=_blank>future version de FreshRSS</a>',
```

File: app/i18n/en/conf.php [hunk 2]
```diff
@@ -48,7 +48,13 @@
 			'timeout' => 'HTML5 notification timeout',
 		),
 		'show_nav_buttons' => 'Show the navigation buttons',
-		'theme' => 'Theme',
+		'theme' => array(
+			'_' => 'Theme',
+			'deprecated' => array(
+				'_' => 'Deprecated',	// TODO
+				'description' => 'This theme is deprecated and will be not available anymore in a <a href=https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#theme target=_blank>future release of FreshRSS</a>',	// TODO
```
Comment (Alkarex) [inline app/i18n/en/conf.php]:
```suggestion
				'_' => 'Deprecated',	// TODO
				'description' => 'This theme is deprecated and will be not available anymore in a <a href="https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#theme" target="_blank">future release of FreshRSS</a>',	// TODO
```

Please add quotes (for all the translations)

Comment (Frenzie) [inline app/i18n/en/conf.php]: Deprecated is programmer jargon, better to paraphrase.
```suggestion
				'description' => 'This theme is no longer supported and will be not available anymore in a <a href="https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#theme" target="_blank">future release of FreshRSS</a>',	// TODO
```

File: app/i18n/fr/conf.php (hunk 1 above)
Comment (Frenzie) [reply app/i18n/fr/conf.php]: Ce thème*s*? :-)

File: docs/en/users/05_Configuration.md [hunk 3]
```diff
@@ -13,16 +13,23 @@ Available languages are: cz, de, en, es, fr, he, it, ko, nl, oc, pt-br, ru, tr,
...
-* *Dark* by **AD**
-* *Flat design* by **Marien Fressinaud**
-* *Origine* by **Marien Fressinaud**
-* *Origine-compact* by **Kevin Papst**
-* *Pafat* by **Plopoyop**
-* *Screwdriver* by **Mister aiR**
-* *Swage* by **Patrick Crandol**
+There’s no accounting for tastes, which is why FreshRSS offers 13 official themes:
+
+| Theme       | designed by                                   | Notes                                                         |
```
Comment (Alkarex) [inline docs/en/users/05_Configuration.md:18]: Let's do the same for https://github.com/FreshRSS/FreshRSS/blob/edge/docs/fr/users/05_Configuration.md
I can help with the translation:

| Thème | Auteur | Notes |

File: app/i18n/fr/conf.php (hunk 1 above)
Comment (Alkarex) [reply app/i18n/fr/conf.php]: Good catch @Frenzie 👍🏻

File: app/i18n/fr/conf.php [hunk 4]
```diff
@@ -48,7 +48,13 @@
 			'timeout' => 'Temps d’affichage de la notification HTML5',
 		),
 		'show_nav_buttons' => 'Afficher les boutons de navigation',
-		'theme' => 'Thème',
+		'theme' => array(
+			'_' => 'Thème',
+			'deprecated' => array(
+				'_' => 'Deprecated',	// TODO
+				'description' => 'This theme is deprecated and will be not available anymore in a <a href="https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#theme" target="_blank">future release of FreshRSS</a>',	// TODO
```
Comment (Alkarex) [inline app/i18n/fr/conf.php]:
```suggestion
				'_' => 'Obsolète',
				'description' => 'Ce thème est obsolète et sera supprimé dans une <a href="https://freshrss.github.io/FreshRSS/fr/users/05_Configuration.html#th%C3%A8me" target=_blank>future version de FreshRSS</a>',
```

File: app/i18n/en-us/conf.php [hunk 5]
```diff
@@ -48,7 +48,13 @@
 			'timeout' => 'HTML5 notification timeout',	// IGNORE
 		),
 		'show_nav_buttons' => 'Show the navigation buttons',	// IGNORE
-		'theme' => 'Theme',	// IGNORE
+		'theme' => array(
+			'_' => 'Theme',	// IGNORE
+			'deprecated' => array(
+				'_' => 'Deprecated',	// TODO
```
Comment (Alkarex) [inline app/i18n/en-us/conf.php]: Needs to be added to the ignore list

File: app/views/configure/display.phtml [hunk 6]
```diff
@@ -71,8 +71,18 @@
 								<?php } ?>
 							</div>
 							<div class="properties">
-								<div><?= sprintf('%s — %s %s', $theme['name'], _t('gen.short.by_author'), $theme['author']) ?></div>
-								<div><?= $theme['description'] ?></div>
+								<div>
+									<?php if (isset($theme['deprecated']) && $theme['deprecated']) { ?>
```
Comment (Alkarex) [inline app/views/configure/display.phtml]:
```suggestion
									<?php if (!empty($theme['deprecated'])) { ?>
```

Minor style preference suggestion. Same below

File: app/views/configure/display.phtml [hunk 7]
```diff
@@ -71,8 +71,18 @@
...
+										<span class="deprecated error"><?= _t('conf.display.theme.deprecated') ?>:</span>
+									<?php } ?>
+									<?= sprintf('%s — %s %s', $theme['name'], _t('gen.short.by_author'), $theme['author']) ?>
+								</div>
+								<div>
+									<?php if (isset($theme['deprecated']) && $theme['deprecated']) { ?>
+										<span class="deprecated"><?= _t('conf.display.theme.deprecated.description') ?></span><br />
+									<?php } ?>
+									<?= $theme['description'] ?>
```
Comment (Alkarex) [inline app/views/configure/display.phtml:84]: Have we checked that this is already HTML-encoded? Please try with `< & > " '` in the description and author name

File: app/i18n/en/conf.php [hunk 8]
```diff
@@ -48,7 +48,13 @@
 			'timeout' => 'HTML5 notification timeout',
 		),
 		'show_nav_buttons' => 'Show the navigation buttons',
-		'theme' => 'Theme',
+		'theme' => array(
+			'_' => 'Theme',
+			'deprecated' => array(
+				'_' => 'Deprecated',	// TODO
```
Comment (Frenzie) [inline app/i18n/en/conf.php]: Btw, this should also say "No longer supported" then :-) (Or is that what the // TODO is saying?)

File: app/i18n/en/conf.php (hunk 2 above)
Comment (math-GH) [reply app/i18n/en/conf.php]: Should we rephrase this too? 
grafik

Comment (Frenzie) [reply app/i18n/en/conf.php]: The same would apply, but "deprecated and will be removed" is much clearer than "deprecated and will be not available anymore".

So `no longer supported and will be removed` is better for both.

File: app/views/configure/display.phtml (hunk 7 above)
Comment (math-GH) [reply app/views/configure/display.phtml:84]: It looks fine:

grafik

grafik

File: app/i18n/en/conf.php (hunk 8 above)
Comment (math-GH) [reply app/i18n/en/conf.php]: For me it is fine to keep it marked as "depricated", because the explanation is in the next line
grafik

File: app/views/configure/display.phtml (hunk 6 above)
Comment (math-GH) [reply app/views/configure/display.phtml]: done

**Comment by math-GH** [comment]
Anything left to do here?

File: app/i18n/fr/conf.php [hunk 9]
```diff
@@ -49,7 +49,13 @@
 			'timeout' => 'Temps d’affichage de la notification HTML5',
 		),
 		'show_nav_buttons' => 'Afficher les boutons de navigation',
-		'theme' => 'Thème',
+		'theme' => array(
+			'_' => 'Thème',
+			'deprecated' => array(
+				'_' => 'Obsolète',
+				'description' => 'Ce thème est obsolète et sera supprimé dans une <a href="https://freshrss.github.io/FreshRSS/fr/users/05_Configuration.html#th%C3%A8me" target="_blank">future version de FreshRSS</a>',
```
Comment (Alkarex) [inline app/i18n/fr/conf.php:56]: `%` signs making the page to crash fixed in https://github.com/FreshRSS/FreshRSS/pull/5022
